src/main/scala/ch/epfl/bluebrain/nexus/kg/config/AppConfig.scala

```diff
@@ -0,0 +1,39 @@
+package ch.epfl.bluebrain.nexus.kg.config
+
+import java.nio.file.Path
+
+import akka.http.scaladsl.model.Uri
+import ch.epfl.bluebrain.nexus.admin.client.config.AdminConfig
+import ch.epfl.bluebrain.nexus.iam.client.IamUri
+import ch.epfl.bluebrain.nexus.kg.config.AppConfig._
+
+import scala.concurrent.duration.Duration
+
+final case class AppConfig(description: Description,
+                           http: HttpConfig,
+                           cluster: ClusterConfig,
+                           persistence: PersistenceConfig,
+                           attachments: AttachmentsConfig,
+                           admin: AdminConfig,
+                           iam: IamConfig)
+
+object AppConfig {
+
+  final case class Description(name: String, environment: String) {
+    def version: String = s"$name-${BuildInfo.version.replaceAll("\\.", "-")}-$environment"
```

Comment:
We need a to do a `replaceAll("\\W", "-")` since we're using `sbt-release-early`